internal/framework/controller/filter/filter_test.go

```diff
@@ -52,8 +51,6 @@ func TestCreateSingleResourceFilter(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			g := NewGomegaWithT(t)
```

Comment:
I think we still need this line here. Without it we will get an error message like " test executed panic(nil) or runtime.Goexit: subtest may have called FailNow on a parent test" when the subtest fails.

Reply:
Ah ok I was confused why we would declare `NewGomegaWithT` inside the test loop, but that makes sense when they are subtests.